remote.go

```diff
@@ -1095,8 +1095,117 @@ func (wd *remoteWD) KeyUp(keys string) error {
 	return wd.keyAction("keyUp", keys)
 }
 
-// TODO(minusnine): Implement PerformActions and ReleaseActions, for more
-// direct access to the W3C specification.
+// KeyPauseAction build a KeyAction which pauses for the supplied duration.
+func KeyPauseAction(duration time.Duration) KeyAction {
+	return KeyAction{
+		"type":     "pause",
+		"duration": uint(duration / time.Millisecond),
+	}
+}
+
+// KeyUpAction build a KeyAction press.
+func KeyUpAction(key string) KeyAction {
+	return KeyAction{
+		"type":  "keyUp",
+		"value": key,
+	}
+}
+
+// KeyDownAction build a KeyAction which presses and holds
+// the specified key.
+func KeyDownAction(key string) KeyAction {
+	return KeyAction{
+		"type":  "keyDown",
+		"value": key,
+	}
+}
+
+// PointerPause build a PointerAction which pauses for the supplied duration.
+func PointerPauseAction(duration time.Duration) PointerAction {
+	return PointerAction{
+		"type":     "pause",
+		"duration": uint(duration / time.Millisecond),
+	}
+}
+
+// PointerMove build a PointerAction which moves the pointer.
+func PointerMoveAction(duration time.Duration, offset Point, origin PointerMoveOrigin) PointerAction {
+	return PointerAction{
+		"type":     "pointerMove",
+		"duration": uint(duration / time.Millisecond),
+		"origin":   origin,
+		"x":        offset.X,
+		"y":        offset.Y,
+	}
+}
+
+// PointerUp build an action which releases the specified pointer key.
+func PointerUpAction(button MouseButton) PointerAction {
+	return PointerAction{
+		"type":   "pointerUp",
+		"button": button,
+	}
+}
+
+// PointerDown build a PointerAction which presses
+// and holds the specified pointer key.
+func PointerDownAction(button MouseButton) PointerAction {
+	return PointerAction{
+		"type":   "pointerDown",
+		"button": button,
+	}
+}
+
+// storedActions an array used to store KeyActions and PointerActions.
```

Comment:
nit: 

// storedActions stores KeyActions and PointerActions for later execution.

(feel free to edit that as you see fit).

Reply:
done